test/couchdb_views_tests.erl

```diff
@@ -286,7 +286,13 @@ couchdb_1309(DbName) ->
         ?assert(is_pid(NewIndexerPid)),
         ?assert(is_process_alive(NewIndexerPid)),
         ?assertNotEqual(IndexerPid, NewIndexerPid),
-        ?assertEqual(2, count_users(DbName)),
+        UserCnt = case count_users(DbName) of
```

Comment:
fine for the positive case but it's an infinite loop if this never happens, and we'd rather the test failed eventually.